packages/aws-client/package.json

```diff
@@ -52,6 +52,7 @@
     "@aws-sdk/client-dynamodb-streams": "^3.447.0",
     "@aws-sdk/client-ec2": "^3.447.0",
     "@aws-sdk/client-ecs": "^3.447.0",
+    "@aws-sdk/client-elasticsearch-service": "^3.529.1",
```

Comment:
Is there a reason to not use the same 3.447? There was some discussion about using the most recent vs. keeping them consistent.

Reply:
I know there's been discussion about this but I'm not sure what benefit that offers us. If you do an `npm install @aws-sdk/client-elasticsearch-service@'^3.447.0' it installs 3.529.1. Given how the v3 SDK is distributed is there a good reason to _not_ install newer versions of the individual clients?

Reply:
Yeah, I don't think there was a _super_ compelling reason to keep them consistent. Doesn't seem like it matters either way as we'll just pull the latest 3.x anyway 🤷 

Someone gave me this feedback so I'm just blindly passing it along 😆 

Makes sense to me to use the latest available at the time of release.